public/api-docs/swagger.js

```diff
@@ -0,0 +1,49 @@
+import swaggerJSDoc from 'swagger-jsdoc';
```

Comment:
remove this page from the public folder.

Reply:
Noted, and done